libtransmission/handshake.cc

```diff
@@ -437,7 +437,7 @@ ReadState tr_handshake::read_crypto_provide(tr_peerIo* peer_io)
         obfuscated_hash[i] = x_or[i] ^ req3[i];
     }
 
-    if (auto const info = mediator_->torrent_from_obfuscated(obfuscated_hash); info)
+    if (auto const info = mediator_->torrent_from_obfuscated(obfuscated_hash); info && info->is_running)
```

Comment:
Slightly misleading log message in the `else` branch now.

Reply:
fa2d688d16855c5144c3154cd9f1a506636fa3a1